index.js

```diff
@@ -86,45 +88,63 @@ exports.init = function (sbot, config) {
     sbot.publish(content, cb)
   }
 
+  function graph (cb) {
+    onReady(() => {
+      cb(null, layered.getGraph())
+    })
+  }
+
+  function graphStream (opts) {
+    opts = opts || {}
+    const live = opts.live === true // default is false
+    const old = opts.old !== false // default is true
```

Comment:
If you're requiring node 10, I think destructuring with default values is a lot cleaner to read

```
const {
  live = false, 
  old = true
} = opts
```

Reply:
Oh interesting. I know about that syntax but I'm not used to using it that often. I could use that. I like the even-newer (not supported in Node 10) style: `const live = opts.live ?? false` but the one you mentioned is also sweet.

Reply:
I like it for when you have lots of opts because you end up with a really nice summary all bundled together